src/main/java/org/dcsa/tnt/model/base/AbstractEventSubscription.java

```diff
@@ -7,25 +7,64 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.relational.core.mapping.Column;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 import java.util.UUID;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class AbstractEventSubscription extends AuditBase {
 
+    public String getBookingReference() {
+        return carrierBookingReference;
+    }
+
+    public void setBookingReference(String bookingReference) {

```

Comment:
Can we remove the `getBookingReference()` and  `setBookingReference(String bookingReference)` ? Unless its been used already, is it?  `@Data` will create the getter and setter.

Reply:
We do not have `BookingReference` as a variable, it was removed in this [commit](url). That is why we have the getters and setters.